### PR TITLE
Add connection labels

### LINKED ISSUE #150: Labels for connections
## Problem
In the moment its not possible to create labels for connections. We need to add the possibilities to add these.

## Requirements
- [x] When clicking on the naming tool from the formalism, it should create a label at the previous by the formalism defined position.
- [x] Loading label should appear at the right position, so it has to be saved within the file.

### PR DESCRIPTION
**Pull request by 0x4d**
Closes #150 

## Describe your changes
- Calculate connection label position after import (needs [renew-formalism-pt#24](https://github.com/renew-js/renew-formalism-pt/pull/24)) 
- Calculate connection label position in CreateLabelBehavior

### CODE REVIEW

File: src/core/Canvas.js [hunk 1]
```diff
@@ -63,6 +63,10 @@ export class Canvas extends BaseCanvas {
         return +this._viewport.getCTM().a.toFixed(5);
     }
 
+    getConnectionBBox (connection) {
```
Comment (tim-kilian) [inline src/core/Canvas.js]: Do this only work for connections ? Could we do a more general method?

File: src/features/create/behaviors/CreateLabelBehavior.js [hunk 2]
```diff
@@ -3,9 +3,10 @@ import { Behavior } from '../../../core/eventBus/Behavior';
 
 export class CreateLabelBehavior extends Behavior {
 
-    constructor (eventBus, commandStack, create, toolbox) {
+    constructor (eventBus, canvas, commandStack, create, toolbox) {
```
Comment (tim-kilian) [inline src/features/create/behaviors/CreateLabelBehavior.js:6]: We need to have core components, which are available from all injected classes without explicit injecting it. An issue could be nice for this.

**Review by tim-kilian — CHANGES_REQUESTED** [review]
Please generalize BBox method before merge.

File: src/core/Canvas.js (hunk 1 above)
Comment (0x4d) [reply src/core/Canvas.js]: Haha mind was focused on connections :laughing: Of course this works for any element.

**Comment by 0x4d** [comment]
> Please generalize BBox method before merge.

Will refactor the method signature tonight

**Review by tim-kilian — APPROVED** [review]

File: src/features/create/behaviors/CreateLabelBehavior.js (hunk 2 above)
Comment (tim-kilian) [reply src/features/create/behaviors/CreateLabelBehavior.js:6]: I've created an issue for this: #156

**Pull request merged**